tests/common/plugins/pdu_controller/snmp_pdu_controllers.py

```diff
@@ -231,7 +231,7 @@ def _get_one_outlet_power(self, cmdGen, snmp_auth, port_id, status):
             current_val = str(val)
             port_oid = current_oid.replace(self.PORT_POWER_BASE_OID, '')
             if port_oid == port_id:
-                status['output_watts'] = current_val
+                status['output_watts'] = 0 if current_val == "" else current_val
```

Comment:
In test_power_off_reboot case, if the default value is 0,  no_power list is not emtpy.
This case will still fail, right?


```
    # If PDU supports returning output_watts, making sure that all outlets has power.
    no_power = [item for item in all_outlets if int(item.get('output_watts', '1')) == 0]
    pytest_assert(not no_power, "Not all outlets have power output: {}".format(no_power))

```